src/front-end/typescript/lib/app/router.ts

```diff
@@ -335,6 +335,17 @@ const router: router_.Router<Route> = {
         };
       }
     },
+    {
+      path: prefixPath("/organizations/:id/team-with-us-terms-and-conditions"),
+      makeRoute({ params }) {
+        return {
+          tag: "orgTWUTerms",
+          value: {
+            orgId: params.id || ""
+          }
+        };
```

Comment:
I know the other routes use the plain object syntax to make ADTs, but I think it's better to use the `adt` function as it's more concise and provides a better explanation of what is being done. It might be worth addressing this in a small refactor separately.

Reply:
I'll change that one here, but good point about the refactor